samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -212,6 +215,21 @@ void BLEConnectivityManager::DisconnectionHandler(bt_conn *conn, uint8_t reason)
 	}
 }
 
+#ifdef CONFIG_BRIDGE_FORCE_BT_CONNECTION_PARAMS
+bool CheckParamChangeRequest(struct bt_le_conn_param *param)
+{
+	/* It is necessary to decrease Bluetooth LE activity to leave some radio time to Thread */
+	return (param->interval_max >= CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MAX &&
```

Comment:
Is this condition correct? I'd expect `<=` here assuming that we return `true` if everything is correct.

Reply:
Yes it is correct we return true when peripheral connection parameters are less demanding than these one we want to force, for example we set conn interval  =  50 ms, peripheral propose conn interval  = 60ms so we accept its parameters.

Reply:
But of course, condition could be changed, like that, if it matters
                              
CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MAX <= param->interval_max

Reply:
I think that our logic is correct, as you mentioned, it's just about providing our bridge the most relaxed scenario.

Reply:
Ok, thx